api/graphql/schema.graphql

```diff
@@ -202,7 +202,8 @@ type EmployeeProfile {
     @rename(attribute: "career_planning_promotion_move_time_frame")
   promotionMoveOrganizationType: [LocalizedOrganizationTypeInterest!]
     @rename(attribute: "career_planning_promotion_move_organization_type")
-
+  learningOpportunitiesInterest: [LocalizedLearningOpportunitiesInterest!]
```

Comment:
I'm guessing your question about being forced to localize the enum is because of using the Localized* type here.  If you switch this to `[LearningOpportunitiesInterest!]` and remove the `HasLocalization` trait from the enum you'll be OK.